internal/exec/workflow_utils.go

```diff
@@ -163,9 +164,9 @@ func ExecuteDescribeWorkflows(
 	for _, f := range files {
 		var workflowPath string
 		if u.IsPathAbsolute(atmosConfig.Workflows.BasePath) {
-			workflowPath = filepath.Join(atmosConfig.Workflows.BasePath, f)
+			workflowPath = path.Join(atmosConfig.Workflows.BasePath, f)
 		} else {
-			workflowPath = filepath.Join(atmosConfig.BasePath, atmosConfig.Workflows.BasePath, f)
+			workflowPath = path.Join(atmosConfig.BasePath, atmosConfig.Workflows.BasePath, f)
```

Comment:
This PR is reverting a change from `path.Join` to → `filepath.Join` due to inadequate testing. We will open a new PR to migrate from `path.Join` to → `filepath.Join`, so that we do not break `main`.